api/client/src/intTest/java/org/projectnessie/client/auth/oauth2/ITOAuth2Client.java

```diff
@@ -292,51 +332,29 @@ void testOAuth2ClientUnauthorizedBadPassword() {
   }
 
   @Test
-  void testOAuth2ClientExpiredToken() {
+  void testOAuth2ClientExpiredToken() throws InterruptedException {
     OAuth2ClientParams params = clientParams("Client1").build();
+    AtomicReference<AccessToken> token = new AtomicReference<>();
     try (OAuth2Client client = new OAuth2Client(params);
-        HttpClient validatingClient = validatingHttpClient("Client1").build()) {
+        NessieApiV1 nessie = nessieApi(NessieApiV1.class, token::get)) {
       Tokens tokens = client.fetchNewTokens();
-      // Emulate a token expiration; we don't want to wait 10 seconds just for the token to really
-      // expire.
-      revokeAccessToken(validatingClient, tokens.getAccessToken());
-      soft.assertThatThrownBy(() -> tryUseAccessToken(validatingClient, tokens.getAccessToken()))
-          .isInstanceOf(HttpClientException.class)
-          .hasMessageContaining("401");
+      token.set(tokens.getAccessToken());
+      // Revoking the token is not enough because Nessie only
+      // validates the JWT token, it doesn't query Keycloak,
+      // so we need to wait until the token is expired :-(
+      Thread.sleep(11000);
```

Comment:
This is slightly frustrating. Another option is to simply get rid of this test, it doesn't really test the OAuth2 client features.

Reply:
Yea, then remove it, if it doesn't test anything that's under our control.